blueprints/n-tier-architecture.html.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Getting Started with N-Tier Architecture
+layout: docs
+nav: firecracker
+author: kcmartin
+date: 2025-08-29
+---
+
+## What is n-tier architecture?
+
+When people talk about “**n-tier architecture**,” they’re describing a way of splitting an app into layers (or “tiers”) that each have a specific job. The “**n**” just means there could be two, three, or more tiers depending on how you slice things.
```

Comment:
Just a random comment - I've never heard of this term before! Always just referred to this as the application stack. TIL